tests/wasmfs/wasmfs_fetch.c

```diff
@@ -47,5 +120,20 @@ int main() {
   printf("%s\n", buf);
   assert(strcmp(buf, "hello, fetch") == 0);
 
+  assert(close(fd) == 0);
+  assert(close(fd2) == 0);
+}
+
+int main() {
+  getUrlOrigin(url_orig, sizeof(url_orig));
+  test_default();
+
+  // NOTE: Each fetch backend runs in a separate thread.
+#ifdef PROXYING
```

Comment:
The problem is that when running on the main thread, this option is passed: `-sPTHREAD_POOL_SIZE=1`. 
Each fetch backend instance requires a separate thread, so the test just hangs because it can't spawn more than one.

Should the setting be updated with the actual thread count or removed at all?
It passes with `-sPTHREAD_POOL_SIZE=4`.